docs/source/whatsnew/pr/simple_prompt.rst

```diff
@@ -0,0 +1,4 @@
+when using IPython as a subprocess, like for emacs inferior-shell, IPython can
+be started with --simple-prompt flag, which will bypass all the prompt_toolkit
```

Comment:
started with the...

Also, 'all the' doesn't scan quite right for a single thing (it needs a plural, as in *all the things*). In general, you can say 'the whole...', but in this case I think it's best just to drop 'all' and say 'bypass the prompt_toolkit input layer'.